.eslintrc

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
let's use the `extends: "gulp"` eslintrc for now.  It requires eslint@^2 though.

Reply:
First I used it but eslint output errors because js-liftoff uses `const`.

Reply:
@sttk yeah, I had to update the `const` to `var` in other modules as well.  If you don't have the time to do that, I can try to do it this weekend.